.gas-snapshot

```diff
@@ -1,50 +1,51 @@
-AddOwnerAddressTest:testEmitsAddOwner() (gas: 92330)
-AddOwnerAddressTest:testIncreasesOwnerIndex() (gas: 90548)
-AddOwnerAddressTest:testRevertsIfAlreadyOwner() (gas: 93093)
-AddOwnerAddressTest:testRevertsIfCalledByNonOwner() (gas: 11732)
-AddOwnerAddressTest:testSetsIsOwner() (gas: 90432)
-AddOwnerAddressTest:testSetsOwnerAtIndex() (gas: 98831)
-AddOwnerPublicKeyTest:testEmitsAddOwner() (gas: 115377)
-AddOwnerPublicKeyTest:testRevertsIfAlreadyOwner() (gas: 116169)
-AddOwnerPublicKeyTest:testRevertsIfCalledByNonOwner() (gas: 11656)
-AddOwnerPublicKeyTest:testSetsIsOwner() (gas: 113698)
-AddOwnerPublicKeyTest:testSetsOwnerAtIndex() (gas: 128444)
-CoinbaseSmartWallet1271InputGeneratorTest:testGetReplaySafeHashForDeployedAccount() (gas: 308642)
-CoinbaseSmartWallet1271InputGeneratorTest:testGetReplaySafeHashForUndeployedAccount() (gas: 291413)
-CoinbaseSmartWalletFactoryTest:testDeployDeterministicPassValues() (gas: 267720)
-CoinbaseSmartWalletFactoryTest:test_CreateAccount_ReturnsPredeterminedAddress_WhenAccountAlreadyExists() (gas: 287384)
-CoinbaseSmartWalletFactoryTest:test_createAccountDeploysToPredeterminedAddress() (gas: 269071)
-CoinbaseSmartWalletFactoryTest:test_createAccountSetsOwnersCorrectly() (gas: 277937)
+AddOwnerAddressTest:testEmitsAddOwner() (gas: 92374)
+AddOwnerAddressTest:testIncreasesOwnerIndex() (gas: 90614)
+AddOwnerAddressTest:testRevertsIfAlreadyOwner() (gas: 93137)
+AddOwnerAddressTest:testRevertsIfCalledByNonOwner() (gas: 11754)
+AddOwnerAddressTest:testSetsIsOwner() (gas: 90476)
+AddOwnerAddressTest:testSetsOwnerAtIndex() (gas: 98897)
+AddOwnerPublicKeyTest:testEmitsAddOwner() (gas: 115509)
+AddOwnerPublicKeyTest:testFuzzIsOwnerPublicKey(bytes32,bytes32) (runs: 256, μ: 115333, ~: 115333)
+AddOwnerPublicKeyTest:testRevertsIfAlreadyOwner() (gas: 116301)
+AddOwnerPublicKeyTest:testRevertsIfCalledByNonOwner() (gas: 11744)
+AddOwnerPublicKeyTest:testSetsIsOwner() (gas: 113775)
+AddOwnerPublicKeyTest:testSetsOwnerAtIndex() (gas: 128543)
+CoinbaseSmartWallet1271InputGeneratorTest:testGetReplaySafeHashForDeployedAccount() (gas: 283108)
+CoinbaseSmartWallet1271InputGeneratorTest:testGetReplaySafeHashForUndeployedAccount() (gas: 265831)
+CoinbaseSmartWalletFactoryTest:testDeployDeterministicPassValues() (gas: 249076)
+CoinbaseSmartWalletFactoryTest:test_CreateAccount_ReturnsPredeterminedAddress_WhenAccountAlreadyExists() (gas: 261498)
```

Comment:
interesting that gas is down here

Reply:
I think this is because clones1967 have some overhead setting the implementation slot? Not 100%